commonItems/UnitTests/CommonFunctionsTests.cs

```diff
@@ -0,0 +1,140 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace commonItems.UnitTests {
+    public class CommonFunctionsTests {
+        [Fact]
+        public void LastDigitOneGivesSt() {
```

Comment:
Find a way to indicate which function each of these is testing. Either make the names fit, or split into multiple test classes.

Reply:
I've split CommonFunctionsTests into multiple test classes.